### PR TITLE
ViewExpertProfile

### PR DESCRIPTION
**Pull request by Z-Ramezani**
ViewExpertProfile page to show expert profile to users.

### CODE REVIEW

File: front/se_project/src/Pages/viewArtistProfile.jsx [hunk 1]
```diff
@@ -49,4 +48,5 @@ const ViewArtistProfile = (props) => {
     );
 }
  
-export default ViewArtistProfile;
```
Comment (Madanian) [inline front/se_project/src/Pages/viewArtistProfile.jsx:52]: this changes is ok

File: front/se_project/src/Pages/viewExpertProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        <div className='d-flex flex-column min-vh-100'>
+            <Navbar/>
+            <Breadcrumbb is3OrNot='true' first='صفحه اصلی' second='آگهی اثر' third='نمایش حساب کاربری کارشناس'/>
+            <HorizontalLine value='اطلاعات فردی'/>
+            <form style={{backgroundColor:'#F9FFFF'}}>
+                <div className='container w-75 rounded'>
+                    <div className='row justify-content-center'>
+                        <div class="row position-relative" dir="LTR"  >
+                            <div class="col-md-5 p-md-4">
+                                    <img class="img-fluid" src={x}/*get the first img from back */ alt="profile image"  style={{blockSize: "9vw", maxWidth:'100%',maxHeight:'100%',marginLeft:'5vw'}}/>
```
Comment (Madanian) [inline front/se_project/src/Pages/viewExpertProfile.jsx:24]: I think in prototype we don't have profile image for expert! I think you should delete this tag.

File: front/se_project/src/Pages/viewExpertProfile.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                                    <h2 class="mt-3" style={{fontSize:'1.1vw', fontWeight:'bold'}}>راه ارتباطی: {/*recive from back*/}</h2>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </form>
+            <HorizontalLine value='سوابق کاری'/>
+            <div className='container w-75 p-3 my-1' style={{marginLeft:'10.5%'}} >
+                <h2  style={{fontSize:'90%', fontWeight:'bold'}}>زمینه فعالیت : {/*recive from back*/}</h2>
+                <h2 className='mt-3' style={{fontSize:'90%', fontWeight:'bold'}}> سابقه فعالیت: {/*recive from back*/}</h2>
```
Comment (Madanian) [inline front/se_project/src/Pages/viewExpertProfile.jsx:40]: add placeholder for this: ماه

File: front/se_project/src/Pages/viewExpertProfile.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+            </div> 
+            <div className='container w-75 p-3 shadow bg-white rounded mb-5'>
+                    <ShowArtworkAdCost/>
+                    <h2 class="mt-0 text-danger" style={{fontSize:'110%', fontWeight:'bold', }}> قیمت پیشنهادی کارشناس: {/*recive from back*/}</h2>
```
Comment (Madanian) [inline front/se_project/src/Pages/viewExpertProfile.jsx:45]: add تومان after that based on prototype

File: front/se_project/src/components/navBar/navLinks.jsx [hunk 5]
```diff
@@ -63,7 +63,7 @@ const NavLinks = (props) => {
                         <ul className="dropdown-menu dropdown-menu" aria-labelledby="navbarDarkDropdownMenuLink">
                         <li><Link class="dropdown-item" to="../../artistProfile" style={{fontSize:'0.8vw'}}>اطلاعات حساب کاربری</Link></li>
                         <li><Link class="dropdown-item" to="../../TicketPart" style={{fontSize:'0.8vw'}}>ثبت آگهی و تیکت</Link></li>
-                        <li><Link class="dropdown-item" to="#" style={{fontSize:'0.8vw'}}>خروج</Link></li>
+                        <li><Link class="dropdown-item" to="../../MainPage" style={{fontSize:'0.8vw'}}>خروج</Link></li>
```
Comment (Madanian) [inline front/se_project/src/components/navBar/navLinks.jsx:66]: ok

File: front/se_project/src/components/navBar/navLinks.jsx [hunk 6]
```diff
@@ -82,10 +82,9 @@ const NavLinks = (props) => {
                         </ul>
                         </li>
                         <li className="nav-item mx-2">
-                        <Link className="nav-link" to="#">|</Link>
+                        <Link className="nav-link disabled" to="#">|</Link>
```
Comment (Madanian) [inline front/se_project/src/components/navBar/navLinks.jsx:85]: I don't understand. why do you add this?

File: front/se_project/src/components/navBar/navLinks.jsx [hunk 7]
```diff
@@ -82,10 +82,9 @@ const NavLinks = (props) => {
...
                         </li>
-                        {/* کارای مسخره زهرا */}
-                        <Link  to="../../viewArtistProfile" className="nav-link">ورود</Link>
+                        <Link  to="../../login" className="nav-link">ورود</Link>
```
Comment (Madanian) [inline front/se_project/src/components/navBar/navLinks.jsx:87]: ok

**Review by Madanian — CHANGES_REQUESTED** [review]
check my comments before merge

File: front/se_project/src/Pages/viewExpertProfile.jsx (hunk 3 above)
Comment (270380) [reply front/se_project/src/Pages/viewExpertProfile.jsx:40]: className in line 25, 29, 30, 31

File: front/se_project/src/Pages/viewExpertProfile.jsx (hunk 4 above)
Comment (270380) [reply front/se_project/src/Pages/viewExpertProfile.jsx:45]: className line 45

File: front/se_project/src/Pages/viewExpertProfile.jsx [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+                    <br/>
+                    <h2 class="mt-0 text-danger" style={{fontSize:'90%', fontWeight:'bold', }}> توضیحات: {/*recive from back*/}</h2>
```
Comment (270380) [inline front/se_project/src/Pages/viewExpertProfile.jsx:47]: className

File: front/se_project/src/Pages/mainPage.jsx [hunk 9]
```diff
@@ -42,6 +42,12 @@ const MainPage = (props) => {
             <Breadcrumbb is1OrNot='true' first='صفحه اصلی'/>
             <div className='d-flex flex-column min-vh-100'>
                 <div className='container w-75 p-3'>
+                    <div class="input-group rounded">
```
[the remark this answers is not in the document]
Comment (Z-Ramezani) [reply front/se_project/src/Pages/mainPage.jsx:45]: done!

File: front/se_project/src/Pages/viewExpertProfile.jsx (hunk 2 above)
Comment (Z-Ramezani) [reply front/se_project/src/Pages/viewExpertProfile.jsx:24]: I can delete the image but in user information there is no info to show.
we can add this.

File: front/se_project/src/Pages/viewExpertProfile.jsx (hunk 3 above)
Comment (Z-Ramezani) [reply front/se_project/src/Pages/viewExpertProfile.jsx:40]: done

File: front/se_project/src/Pages/viewExpertProfile.jsx (hunk 4 above)
Comment (Z-Ramezani) [reply front/se_project/src/Pages/viewExpertProfile.jsx:45]: we can add تومان as placeholder

Comment (Z-Ramezani) [reply front/se_project/src/Pages/viewExpertProfile.jsx:45]: done

File: front/se_project/src/components/navBar/navLinks.jsx (hunk 6 above)
Comment (Z-Ramezani) [reply front/se_project/src/components/navBar/navLinks.jsx:85]: disable to invariant the shape of mouse

**Review by Z-Ramezani — COMMENTED** [review]
checked!

**Pull request merged**